src/js/views/validator.js

```diff
@@ -0,0 +1,18 @@
+import { ERROR_MESSAGE, priceMax, priceMin, pricePerUnit } from "../utils/const.js";
+
+export class Validator {
+    constructor() {}
+
+    isValidated = (price) => {
+        if (!price) return this.#afterValidate(ERROR_MESSAGE.InputRequired);
+        if (price < priceMin) return this.#afterValidate(ERROR_MESSAGE.InputMinInsufficient);
+        if (price > priceMax) return this.#afterValidate(ERROR_MESSAGE.InputMaxExceeded);
+        if (!!(price % pricePerUnit)) return this.#afterValidate(ERROR_MESSAGE.IncorrectUnit);
```

Comment:
```suggestion
        if (price % pricePerUnit !== 0) return this.#afterValidate(ERROR_MESSAGE.IncorrectUnit);
```
!!에 의존하는 것 보다 0이 아닐때라는점을 알려주는게 더 직관적으로 보여요~